src/app/story/storyrow/storyrow.component.css

```diff
@@ -162,3 +167,7 @@
     /* Hides the shadow effect of node's img while it is being bordered */
     opacity: 0;
 }
+
+.total-time-column {
```

Comment:
This should be plural to match the rest `total-times-column`. This should also be in the "Blocks" section.